### ISSUE TITLE
ABANDONED?

### ISSUE THREAD

**Issue by huglester** [phpWhois/phpWhois#89]
Hello,

seems no PRs are accepted, etc.
Maybe you can share the 'new' goto fork, or similar package?

Thank you

**Comment by adamjimenez** [comment]
I've started contributing patches to: https://github.com/jsmitty12/phpWhois
This fork supports PHP 7+

**Comment by huglester** [comment]
I started earlier mergin different patches from different forks...
https://github.com/huglester/phpWhois

but will look maybe I just push all updates there. or just merge his updates to my fork.

anyway thanks

**Comment by tomsommer** [comment]
Decide please :)

**Comment by djvu61** [comment]
im not sure wich one to choose

**Comment by djvu61** [comment]
+1 for jsmitty12/phpwhois

**Comment by kevinoo** [comment]
Yes, I think this repo is abandoned. Since 2023, [jsmitty12](https://github.com/jsmitty12) has also abandoned his repository.

I took this repository by the hand, creating a new fork [packagist: kevinoo/phpwhois](https://packagist.org/packages/kevinoo/phpwhois).

**Comment by kevinoo** [comment]
Hi @huglester, @adamjimenez, @tomsommer, @djvu61 

I think this repo is officially abandoned. Since 2023, [jsmitty12](https://github.com/jsmitty12) has also abandoned his repository.

I took this repository by the hand, creating a new fork [packagist: kevinoo/phpwhois](https://packagist.org/packages/kevinoo/phpwhois).
